src/backup.ts

```diff
@@ -178,6 +181,10 @@ class Backup {
         name: this.formattedName_,
       },
     };
+    if ('versionTime' in options) {
+      reqOpts.backup!.versionTime =
+        Spanner.timestamp(options.versionTime).toStruct();
+    }
```

Comment:
PTAL this file since I had to add hand-written code to support `versionTime` when creating a backup.

Reply:
LGTM. Should we add `versionTime` to the `@example` in the comments?

Reply:
Good idea. Done.